ci-operator/step-registry/upi/conf/gcp/upi-conf-gcp-commands.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+### TODO: Move this to Dockerfile.upi.ci
+export HOME=/tmp
+pip install --user pyyaml
+
+pushd ${SHARED_DIR}
+
+### Empty the compute pool (optional)
+python -c '
+import yaml;
+path = "install-config.yaml";
+data = yaml.full_load(open(path));
+data["compute"][0]["replicas"] = 0;
+open(path, "w").write(yaml.dump(data, default_flow_style=False))'
+
+### Create manifests
+cp ${SHARED_DIR}/install-config.yaml ${SHARED_DIR}/install-config.yaml.bak
+openshift-install --dir="${SHARED_DIR}" create manifests 2>&1 | grep --line-buffered -v password &
```

Comment:
> grep --line-buffered -v password &

i don't think we need this for manifests

Reply:
removed.